CHANGELOG/1.2-CHANGELOG.md

```diff
@@ -459,7 +459,7 @@ Relevant PRs:
 * Features have been re-arranged to a struct containing metadata and feature
   Name. The API Channel has been set as a field for such a struct in
   [#3287](https://github.com/kubernetes-sigs/gateway-api/pull/3287)
-* Exposing grpc client interface as a option in the conformance suite in
```

Comment:
I think these entries came from the PR release-note, so even being wrong I am not sure I would change anything on Changelogs

Reply:
Each project differs on this. I can easily drop if you prefer. Personally, as the rare person who reads release notes / searches through them later, I prefer being able to search for and find things / be able to understand them.